manim/mobject/coordinate_systems.py

```diff
@@ -367,6 +362,21 @@ def construct(self):
 
         return line_graph
 
+    def origin_shift(self, axis_range):
+        """Shifts graph mobjects to compensate when 0 is not on the axis.
+
+        Parameters
+        ----------
+        axis_range : Union[:class:`list`, :class:`numpy.ndarray`]
+            The range of the axis : `(x_min, x_max, x_step)`.
+        """
```

Comment:
Typing + some minor corrections. I believe `List` might have to be imported

```suggestion
    def origin_shift(self, axis_range:List[float]):
        """Shifts graph mobjects to compensate when 0 is not on the axis.

        Parameters
        ----------
        axis_range
            The range of the axis : ``(x_min, x_max, x_step)``.
        """
```